### DISCUSSION TITLE
Шрифты

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2_classic#80] [General]
Нужно сравнить все шрифты оригинала и аддона. Как я помню, они отличаются и стилем, и цветом.

**Comment by Kor-Angar** [comment]
шрифты Акеллы?

**Comment by dosinabox** [comment]
Да.

**Comment by Kor-Angar** [comment]
сейчас помогал поставить на стим версию, оказалось что нет шрифтов. пришлось дополнительно скачать шрифты мухомора и установить
![image](https://user-images.githubusercontent.com/49514764/142274574-7ff68283-d5ab-432b-9426-41cdcbc5a8f7.png)
![image](https://user-images.githubusercontent.com/49514764/142274604-963e33dc-45aa-40bb-89ea-358101a6e5d1.png)

**Reply by dosinabox** [reply]
А почему не стим фикс? Там всё есть, да и ещё с Юнионом. СП так-то устарел.

**Reply by Kor-Angar** [reply]
у него Yong Bek со стимфиксом игра не запускалась. он мне писал, потом тебе писал. потом скачал какую то странную сборку с торрента где готика 2 классик но не твоя сборка, а черт пойми что  и опять стал писать.

и вот сегодня я в ручном режиме(удаленно) всё  ему установил на стимверсию Готики 2 НВ: 

- оф. патч пираней
- плэеркит, 
- SystemPack, 
- g2_steam_starter, 
- патч на 4 гб 

и игра пошла.  тогда поставил неоф. обновление НВ и готика 2 классик. в неоф. обновление с шрифтами было всё ок, а в классик не ок. поэтому пришлось еще и скачать шрифты мухомора.

***

у меня тоже с юнионом были какие то глюки, но я не смог их(глюки) отловить что бы Gratt  написать. может из-за виндоус 7.

**Comment by Kor-Angar** [comment]
а ты добавляла улучшенные шрифты акеллы которые на wop.ru выложили?

**Comment by dosinabox** [comment]
[Эти](https://worldofplayers.ru/threads/42479/)? Нет. В моде вообще нет никаких шрифтов.

**Reply by Kor-Angar** [reply]
ага. эти. а почему шрифтов нет? ведь в стимовской версии вроде нет русских шрифтов.

**Reply by dosinabox** [reply]
В стим-фиксе есть, а он требуется для стартера модов.

**Reply by Kor-Angar** [reply]
это который стим-фикс со звуком русским?

**Reply by Kor-Angar** [reply]
то есть который много весит? :-)

**Reply by dosinabox** [reply]
Да, но русская озвучка не обязательна.
В целом, шрифты можно и добавить, раз теперь они есть вычищенные и исправленные.

**Reply by Kor-Angar** [reply]
это я уточняю потому что играю с шрифтами от Мухомора, а твой очень большой стим-фикс не ставил, 
так как ставлю твой иностранный стим фикс (потому что он мало весит), потом мод, а звук или немецкий или пересобранный архив с русским

**Reply by Kor-Angar** [reply]
> В целом, шрифты можно и добавить

это хорошее дело :-)  тогда можно будет просто на стим версию накатить лёгкий стим-фикс, поставить мод и играть :-)

**Reply by dosinabox** [reply]
А по цвету они подходят к классике?

**Reply by Kor-Angar** [reply]
с cd диска 1
[Textures_Fonts_Apostroph.zip](https://github.com/dosinabox/g2_classic/files/7993510/Textures_Fonts_Apostroph.zip)

**Reply by Kor-Angar** [reply]
NOMIP

[NOMIP.zip](https://github.com/dosinabox/g2_classic/files/7993516/NOMIP.zip)

**Reply by Kor-Angar** [reply]
Кириллица
[nomip.zip](https://github.com/dosinabox/g2_classic/files/7993523/nomip.zip)

**Comment by Kor-Angar** [comment]
на всякий случай Gothic2_Akella_Covers
[Gothic2_Akella_Covers.zip](https://github.com/dosinabox/g2_classic/files/7993487/Gothic2_Akella_Covers.zip)

**Comment by Kor-Angar** [comment]
а что то у меня кракозябры какие то в польской версии

![image](https://user-images.githubusercontent.com/49514764/186446233-5b1a6b0c-cc4a-43df-9c7c-782611dddf54.png)

**Reply by dosinabox** [reply]
Версия польская, а шрифты от Мухомора остались кириллические.

**Reply by Kor-Angar** [reply]
![image](https://user-images.githubusercontent.com/49514764/186451001-6a7a44df-0148-4de7-8b96-477d22e7fccc.png)
тут?

**Reply by dosinabox** [reply]
Тут. И если игра запускается через лаунчер стима, то ещё в подписанных модах из мастерской.

**Reply by Kor-Angar** [reply]
![image](https://user-images.githubusercontent.com/49514764/186458826-ce7de2a5-116a-44f3-a915-693bc19048b4.png)
а вот это откуда могло взяться?

**Reply by Kor-Angar** [reply]
![image](https://user-images.githubusercontent.com/49514764/186459654-149e4bfc-33fa-45cd-87fe-e5a76585fc3e.png)
а шрифты мухомора отключены
